contracts/bonding/BondingManager.sol

```diff
@@ -568,7 +571,9 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         uint256 currentRound = roundsManager().currentRound();
         Delegator storage del = delegators[_delegator];
         
-        require(_endRound <= currentRound && _endRound > del.lastClaimRound, "end round must be before or equal to current round and after lastClaimRound");
+        // End round must be before or equal to current round and after lastClaimRound
```

Comment:
Let's remove this comment since the revert reasons below already explain what the require statements do.

Reply:
fixed in 2eb6a0f